config/locales/ca.yml

```diff
@@ -14,27 +31,41 @@ ca:
     previous: "Anterior"
     next: "Següent"
     download: "Descarregar:"
-    has_many_new: "Afegir %{model}"
+    has_many_new: "Afegir un altre %{model}"
     has_many_delete: "Eliminar"
     has_many_remove: "Treure"
+    move: "Moure"
     filters:
       buttons:
-        filter: "Filtrar"
-        clear: "Treure filtres"
+        filter: "Filtra"
+        clear: "Elimina els filtres"
+      predicates:
+        from: "Des de"
+        to: "Fins"
+    scopes:
+      all: "Tots"
+    search_status:
+      title: "Cerca activa"
+      title_with_scope: "Cerca activa per %{name}"
+      no_current_filters: "Sense filtres actius"
     status_tag:
       "yes": "Sí"
       "no": "No"
-      "unset": "No"
-    logout: "Desconnecta't"
+      "unset": "Desconegut"
+    logout: "Tanca la sessió"
     powered_by: "Powered by %{active_admin} %{version}"
     sidebars:
       filters: "Filtres"
+      search_status: "Estat de la cerca"
     pagination:
-      empty: "No hi ha %{model}"
-      one: "S'està mostrant <b>1</b> %{model}"
-      one_page: "S'estan mostrant <b>tots %{n}</b> %{model}"
-      multiple: "S'estan mostrant %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b> de <b>%{total}</b> en total"
-      multiple_without_total: "S'estan mostrant %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b>"
+      empty: "No s'ha trobat cap %{model}"
+      one: "<b>1</b> %{model} de <b>1</b>"
+      one_page: "%{model}, tots %{n}</b>"
+      multiple: "%{model}, <b>%{from}-%{to}</b> d'un total de <b>%{total}</b>"
+      multiple_without_total: "%{model}, <b>%{from}-%{to}</b>"
```

Comment:
These don't follow what we do currently for the `en` locale. Only the `empty` key uses the `model` parameter. You can see a [diff of the pagination keys in the UPGRADING guide](https://github.com/activeadmin/activeadmin/blob/master/UPGRADING.md#localization-updates).

Reply:
Oh yes, sorry again to give you so much work...

Reply:
It's ok! No worries. Thank you. Just one line here needs to be updated. I'll submit another review.